gsdk/src/api.rs

```diff
@@ -23,11 +23,14 @@ use anyhow::Result;
 use core::ops::{Deref, DerefMut};
 use subxt::OnlineClient;
 
+const DEFAULT_GEAR_ENDPOINT: &str = "wss://rpc.vara.network:443";
+const DEFAULT_TIMEOUT: u64 = 60_000;
```

Comment:
If I'm not mistaken the timeout is in milliseconds. It would be nice to mention it in comment or add suffix like `_MILLISECS`

Reply:
[b100cd3](https://github.com/gear-tech/gear/pull/4225/commits/b100cd3b7599cfdbb579d13cb11dad823b90271d) updated, btw I just changed the default `retires` to `0` otherwise it could be unexpected from users' end